### DISCUSSION TITLE
Improve version checking

### DISCUSSION THREAD

**Discussion by Andre601** [Ghost-chu/QuickShop-Reremake#1542] [Ideas]
QuickShop will treat a Snapshot (i.e. `5.0.0.3-SNAPSHOT`) as if it was an outdated version and send the "New version available" message regardless if that is true or not.

My proposal is to make QuickShop remove the `-SNAPSHOT` part from the version and then use that to compare with the version on Spigot. If it's newer or equal, skip, but if it isn't send a notification.

**Comment by Ghost-chu** [comment]
Snapshot branch use Snapshot channel, snapshot update really fast (5+ updates per day), so always says "New version available".

Once QuickShop build under release mode, it will switch to release channel and only check updates for released versions.
